tests/std/tests/P2442R1_views_chunk/test.cpp

```diff
@@ -35,6 +35,9 @@ constexpr bool test_one(Rng&& rng, Expected&& expected) {
     STATIC_ASSERT(forward_range<R> == forward_range<Rng>);
     STATIC_ASSERT(bidirectional_range<R> == bidirectional_range<Rng>);
     STATIC_ASSERT(random_access_range<R> == random_access_range<Rng>);
+    // Validate non-default-initializability
+    STATIC_ASSERT(!is_default_constructible_v<R>);
+    STATIC_ASSERT(!default_initializable<R>);

```

Comment:
A type that cannot be value-initialized certainly cannot be default-initialized, so `!is_default_constructible_v<T>` implies `!default_initializable<T>` - we don't need to redundantly check both. I'll cleanup.